### PR TITLE
Vendor syft/migrate witness -> syft direct

### PR DESCRIPTION
**Pull request by jeff-mccoy**
Adds `zarf tools syft` command for full access to https://github.com/anchore/syft.  Also migrates from 3rd party wrapper (Wtiness) to direct Syft integration.  Refactored code to prepare for adding file-based SBOM generation in addition to image-based.

### CODE REVIEW

**Comment by jeff-mccoy** [comment]
Screen Shot 2022-09-07 at 3 31 16 AM

**Review by Racer159 — APPROVED** [review]
Tested locally and this worked fine.  Like that we are moving to a builder pattern and just fixed a small spelling mistake.

File: src/cmd/tools.go [hunk 1]
```diff
@@ -109,4 +110,13 @@ func init() {
...
 	registryCmd.AddCommand(craneCmd.NewCmdCopy(&cranePlatformOptions))
 	registryCmd.AddCommand(craneCmd.NewCmdCatalog(&cranePlatformOptions))
+
+	syftCmd, err := cli.New()
+	if err != nil {
+		message.Fatal(err, "Unable to create syft CLI")
+	}
+	syftCmd.Use = "syft"
+	syftCmd.Short = "SBOM tools provided by Anchore Syft"
+	syftCmd.Aliases = []string{"s"}
```
Comment (Racer159) [inline src/cmd/tools.go:120]: Do we want to include an alias called "sbom" (i.e. `zarf tools sbom ...`) as well so that it is more obvious what this is?

**Review by Racer159 — CHANGES_REQUESTED** [review]

File: src/cmd/tools.go (hunk 1 above)
Comment (jeff-mccoy) [reply src/cmd/tools.go:120]: Yes we do ❤️ feel free to push the change

**Comment by jeff-mccoy** [comment]
In fact sbom might make more sense and syft as the alias as we did with k9s iirc

**Comment by jeff-mccoy** [comment]
Awesome, thanks for finishing that up.

**Comment by Racer159** [comment]
I fixed the root examples but we may want to fix the sub command examples as well (looks like this would require looping through the cobra child commands and fixing them manually)

**Comment by jeff-mccoy** [comment]
I think elsewhere we just didn't include the vendored-in examples, I think it might be fine to leave them off as it's more a power-user feature and not really our commands anyway.

**Review by Racer159 — APPROVED** [review]

**Comment by Racer159** [comment]
Removed the sub command examples.  Left the high-level examples just as a quick start for folks

**Pull request merged**